jmx-metrics/docs/target-systems/activemq.md

```diff
@@ -0,0 +1,79 @@
+# ActiveMQ Metrics
+
+The JMX Metric Gatherer provides built in ActiveMQ metric gathering capabilities.
+These metrics are sourced from: https://activemq.apache.org/jmx
+
+### Metrics
+
+* Name: `activemq.consumer.count`
+* Description: The number of consumers currently reading from the broker.
+* Unit: `{consumers}`
```

Comment:
nit, is `consumers` ok or are the braces meaningful?

Reply:
not necessarily, that seems easy enough